src/java/io/bazel/rulesscala/scalac/CompileOptions.java

```diff
@@ -29,6 +29,7 @@
   public final String currentTarget;
   public final String statsfile;
   public final String dependencyTrackingMethod;
+  public final /*@Nullable*/ String diagnosticsFile;
```

Comment:
It will probably stay whenever I implement opt-in, but I'll make sure to take a look at this when I do implement, just waiting on some feedback on how it would be preferable to do it 😁

Reply:
It's no longer Nullable due to the opt-in implementation